render_configs.sh

```diff
@@ -1,19 +1,21 @@
+#!/bin/bash
```

Comment:
I changed this file as much as I did because the previous version didn't appear to be used anywhere:

- When run locally, the original script always fails, because it's trying to write configs to a location that doesn't exist in this project
- When deployed in real environments, the configs are passed via deployment infrastructure, not a locally rendered file

Reply:
The script didn't work for me as is. I was getting the following error message 
```
Couldn't start vault with IPC_LOCK. Disabling IPC_LOCK, please use --cap-add IPC_LOCK
```
I was able to fix it by adding `--cap-add=IPC_LOCK` flag to the `docker run` command.

An alternative would be to use the `dsde-toolbox` to access vault instead. You could replace `VAULT_COMMAND` with
```
docker run --rm -i -e VAULT_TOKEN="${VAULT_TOKEN}" broadinstitute/dsde-toolbox:dev vault read
```

Reply:
It showed the same message for me, but still ran successfully. I assumed it was related to having low available resources available on my machine.

Switching to the second suggestion since it seems a bit cleaner.

Reply:
🤦  I was so concerned by the error message I didn't even think to check if it worked (it did). Oh well... thanks for making the change anyways